src/server/dfly_main.cc

```diff
@@ -609,6 +610,16 @@ void sigill_hdlr(int signo) {
   exit(1);
 }
 
+void PrintBasicUsageInfo() {
+  const string flags_log_dir = FLAGS_log_dir.CurrentValue();
+  const string log_dir = (flags_log_dir == string("")) ? string("/tmp/") : flags_log_dir;
+  std::cout << "* Logdir is: " + log_dir << '\n';
+  std::cout << "* For the available flags type dragonfly [--help | --helpfull]\n";
+  const string df_docs = "https://www.dragonflydb.io/docs";
```

Comment:
Not that it matters a great deal, but why not just output it as part of the below statement?